### ISSUE TITLE
$meteor.object should accept CollectionFS collection

### ISSUE THREAD

**Issue by epotvin** [Urigo/angular-meteor#282]
As with https://github.com/Urigo/angular-meteor/issues/72, AngularMeteorObject doesn't accept CFS collection as input. The same fix should be applied in the first place, then every functions like save or delete should be tested and eventually fixed, since they may not be compatibles.

**Comment by epotvin** [comment]
I'm not sure I did it the best possible way, but it seems to work fine on almost any point. I tried to avoid coupling the code with CFS concepts. I updated my example application (https://github.com/epotvin/thumbnail-example) to comply with my last commit and every crud functions work, except when we have custom properties on the document that are not auto updated. 

I added a "description" property to my "image" collection with a binded field on the page. When auto update is set, everything is ok, the value is automatically changed in the database. But when it is set to false, it can't even be changed on the screen. Actually it is set back to the current value instantly. I spent 2 hours trying to find the reason but I didn't.

If the false parameter is removed from this line : https://github.com/epotvin/thumbnail-example/blob/master/client/example.js#L43, everything is fine.

Setting it on collection doesn't seems to change anything.
